NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -30,4 +44,60 @@ std::vector<GlobalIndexType> getIndices(
     return indices;
 }
 
+double norm(GlobalVector const& x, unsigned const global_component,
+            MathLib::VecNormType norm_type,
+            LocalToGlobalIndexMap const& dof_table, MeshLib::Mesh const& mesh)
+{
+    // TODO that also includes ghost nodes.
+    std::size_t const n_nodes = mesh.getNumberOfNodes();
+
+    // TODO for PETSc some global accumulation is necessary.
```

Comment:
@norihiro-w Thanks for the info. I'd like to make the PETSc information later on. I'm a bit skeptical concerning the index set, because it takes additional storage and one has to build it first of all. (Maybe a manual gather is not more complicated)